apis/nucleus/src/__tests__/viz.test.js

```diff
@@ -232,6 +243,27 @@ describe('viz', () => {
     });
   });
 
+  describe('toggleDataView', () => {
```

Comment:
Potentially one more when you specify the state (true/false) and it actually changes? Or is that overkill?